ReactApp/src/utils/rest-api-call.js

```diff
@@ -39,6 +40,11 @@ export const serverPath = {
     REFRESH_TOKEN: 'refreshJWT',
 }
 
+
+export const pathToReturnTypeMap={
+  [serverPath.VIEW_TASKS]: {tasks:[{finished:PRIMITIVE_TYPES.BOOLEAN, }]},
+}
```

Comment:
What is that? Where did you use it?

Reply:
My bad I forgot to change it, anyway it is more orginize way to follow on the api expected return types

Reply:
Please check it now